### PR TITLE
Feature/content updates

### PR DESCRIPTION
**Pull request by KevinEtchells**
## Context

A number of updates needed for the next round of deploys.

## Changes proposed in this pull request

* Some small content changes around the move to DSIT
* Adding a third quote to the homepage
* Allowing videos to be added to blogs
* Allowing draft blogs on Contentful to be viewed locally
* Allowing for a longer summary on the main blog page, separate to the shorter summary for the blogs hub page
* Adding accessible text descriptions to videos

## Guidance to review

* Please check everything looks okay and the code changes look sensible.

### CODE REVIEW

File: src/index.njk [hunk 1]
```diff
@@ -93,6 +93,12 @@ templateEngineOverride: njk
             <footer class="mt-3">House of Commons, Science, Innovation and Technology Committee</footer>
           </blockquote>
         </div>
+        <div class="carousel-item">
+          <blockquote class="iai-homepage-quote">
+            <p class="text-xl">“Artificial intelligence technologies offer the potential to transform the UK’s economic landscape and improve people’s lives across the country”</p>
+            <footer class="mt-3">David Knott <span aria-hidden="true">|</span> Chief Technology Officer <span aria-hidden="true">|</span> UK Government</footer>
```
Comment (rachaelcodes) [inline src/index.njk:99]: It's a relatively minor thing (especially if the content has been signed-off), but the sources seem to be cited in a different format for each quote. If nothing else, I wonder if changing the second to `House of Commons | Science, Innovation and Technology Committee` would seem more consistent

Comment (KevinEtchells) [reply src/index.njk:99]: I _think_ it's one committee - i.e. "The House of Commons Science, Innovation and Technology Comittee". If so, it doesn't make sense to split it up. But if I'm wrong on that, then it does make sense to change!
Whereas the others have clear separations of person, role, organisation, date.

Comment (rachaelcodes) [reply src/index.njk:99]: That makes sense. I think we might need to get rid of the comma after 'Commons', for readability

**Review by rachaelcodes — APPROVED** [review]
Changes all look good

**Pull request merged**